spec/core/data_structures.md

```diff
@@ -278,297 +312,49 @@ Valid Light Client Attack Evidence encompasses three types of attack and must ad
-themselves. This is because transaction hashes are regularly used as identifiers for
-transactions.
-
-### ValidatorsHash
-
-```go
-block.ValidatorsHash == MerkleRoot(state.Validators)
-```
-
-MerkleRoot of the current validator set that is committing the block.
-This can be used to validate the `LastCommit` included in the next block.
-Note that before computing the MerkleRoot the validators are sorted
-first by voting power (descending), then by address (ascending).
-
-### NextValidatorsHash
-
-```go
-block.NextValidatorsHash == MerkleRoot(state.NextValidators)
-```
-
-MerkleRoot of the next validator set that will be the validator set that commits the next block.
-This is included so that the current validator set gets a chance to sign the
-next validator sets Merkle root.
-Note that before computing the MerkleRoot the validators are sorted
-first by voting power (descending), then by address (ascending).
-
-### ConsensusHash
-
-```go
-block.ConsensusHash == state.ConsensusParams.Hash()
-```
-
-Hash of the protobuf-encoding of a subset of the consensus parameters.
-
-### AppHash
-
-```go
-block.AppHash == state.AppHash
-```
-
-Arbitrary byte array returned by the application after executing and commiting the previous block. It serves as the basis for validating any merkle proofs that comes from the ABCI application and represents the state of the actual application rather than the state of the blockchain itself.
-
-The first block's `block.Header.AppHash` is given by `ResponseInitChain.app_hash`.
-
-### LastResultsHash
-
-```go
-block.LastResultsHash == MerkleRoot([]ResponseDeliverTx)
-```
-
-`LastResultsHash` is the root hash of a Merkle tree built from `ResponseDeliverTx` responses (`Log`,`Info`, `Codespace` and `Events` fields are ignored).
-
-The first block has `block.Header.ResultsHash == MerkleRoot(nil)`, i.e. the hash of an empty input, for RFC-6962 conformance.
-
-## EvidenceHash
-
-```go
-block.EvidenceHash == MerkleRoot(block.Evidence)
-```
-
-MerkleRoot of the evidence of Byzantine behaviour included in this block.
-
-### ProposerAddress
-
-```go
-block.Header.ProposerAddress in state.Validators
-```
-
-Address of the original proposer of the block. Must be a current validator.
-
-## Txs
-
-Arbitrary length array of arbitrary length byte-arrays.
-
-## LastCommit
-
-The first height is an exception - it requires the `LastCommit` to be empty:
-
-```go
-if block.Header.Height == state.InitialHeight {
-  len(b.LastCommit) == 0
-}
-```
-
-Otherwise, we require:
-
-```go
-len(block.LastCommit) == len(state.LastValidators)
-
-talliedVotingPower := 0
-for i, commitSig := range block.LastCommit.Signatures {
-  if commitSig.Absent() {
-    continue
-  }
-
-  vote.BlockID == block.LastBlockID
-
-  val := state.LastValidators[i]
-  vote.Verify(block.ChainID, val.PubKey) == true
-
-  talliedVotingPower += val.VotingPower
-}
-
-talliedVotingPower > (2/3)*TotalVotingPower(state.LastValidators)
-```
+## LightBlock
```

Comment:
Maybe a short note on what it is and why we have it i.e. LightBlock is the core data structure used by light clients.